### DISCUSSION TITLE
Add ability to see parent work item title

### DISCUSSION THREAD

**Discussion by ljfraney** [microsoft/azure-boards-estimate#156] [Ideas]
Often, our user stories don't have all of the context needed to estimate. In these cases, that missing context exists in the parent feature. Granted, we should be adding these details to the story so that it isn't necessary to look at the parent. However, it would be nice to be able to see the parent work item's title when estimating.

- Feature: Add login page to the Widget application
  - Story: Create the user interface
  - Story: Add the business logic code
  - Story: Update the documentation
  
If the title of the parent work item were visible, you would be able to see that these stories are specific to the Widget application without switching to a different view in ADO.

**Comment by jjondle** [comment]
Upvoted. We are having the exact same issue.

**Comment by mf-berlin** [comment]
Upvoted!

Requested it too before i saw this discussion in https://github.com/microsoft/azure-boards-estimate/issues/255 and it was tagged as enhancement.
